crates/iota-config/src/genesis.rs

```diff
@@ -628,3 +633,113 @@ impl TokenDistributionScheduleBuilder {
         schedule
     }
 }
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct ValidatorAllocation {
```

Comment:
Maybe we can have minor description here of what is the purpose of this struct?

Reply:
Done